test/checks/keyboard/banner-is-top-level.js

```diff
@@ -0,0 +1,89 @@
+describe('banner-is-top-level', function () {
+	'use strict';
+
+	var fixture = document.getElementById('fixture');
+
+	var checkSetup = axe.testUtils.checkSetup;
+	var shadowSupported = axe.testUtils.shadowSupport.v1;
+
+	afterEach(function () {
+		fixture.innerHTML = '';
+	});
+
+	it('should return false if banner landmark is in main element', function() {
+		var main = document.createElement('main');
+		var banner = document.createElement('div');
+		banner.setAttribute('role','banner');
+		main.appendChild(banner);
+		fixture.appendChild(main);
+		assert.isFalse(checks['banner-is-top-level'].evaluate(banner));
+	});
+
+	it('should return false if banner landmark is in main element', function () {
```

Comment:
This test is an exact duplicate of the one above it. (I've done that before when copy/pasting tests to create new ones. oops)